fares-data-build-tool/create-data
# feature: Fix precommit hooks

## Description

The precommit hooks (the husky script which runs automatically when you run a commit locally) currently fails. 

## Testing instructions

try a commit in the different folders: netex, admin and site then see if lerna notices the changes

You may need to install lerna globally `npm install lerna -g`
